contributing/chestertons-fence.md

```diff
@@ -0,0 +1,79 @@
+# Chesterton's Fence
+
+Hello :wave:
+You're probably here becomes someone directed you to this post in response to you suggesting an idea about how Exercism could be improved.
+Before our team invest time into exploring your idea, they're hoping you'll read this and consider whether your idea might of being thought of before, what pitfalls it might contain.
+**By adding those problems and pitfalls to your suggestion, and considering why your idea might not have already been implemented, you are likely to get a faster and more positive response.**
```

Comment:
This refers to "those problems" but no problems are mentioned other than the pitfalls.

Reply:
```suggestion
**By adding those considerations and potential pitfalls to your suggestion, and considering why your idea might not have already been implemented, you are likely to get a faster and more positive response.**
```